modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/IgniteSnapshotManager.java

```diff
@@ -648,6 +649,22 @@ public boolean isSnapshotCreating() {
             if (!clusterState.hasBaselineTopology())
                 throw new IgniteException("Snapshot operation has been rejected. The baseline topology is not configured for cluster.");
 
+            if (cctx.kernalContext().clientNode()) {
```

Comment:
What happens if the client will be disconnected? `U.oldest(..)` can return null.

Reply:
Or can we get a situation when there are no servers nodes in cluster but client connected to daemon?

Reply:
Fixed.